pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -231,6 +231,36 @@ func mergeDataMaps(in map[string][]byte, maps ...map[string][]byte) map[string][
 	return out
 }
 
+func (c *controller) syncMachineNameToNode(ctx context.Context, machine *v1alpha1.Machine) (machineutils.RetryPeriod, error) {
+	node, err := c.nodeLister.Get(getNodeName(machine))
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Don't return error so that other steps can be executed.
+			return machineutils.LongRetry, nil
+		}
+		klog.Errorf("Error occurred while trying to fetch node object - err: %s", err)
+		return machineutils.LongRetry, err
```

Comment:
LongRetry is too long of a time to wait for these errors.

```suggestion
		return machineutils.ShortRetry, err
```